test/rectangle-test.js

```diff
@@ -0,0 +1,36 @@
+var assert = require('chai').assert;
+var rectangleBits = require('../rectangle')
+
+describe("it can measure a rectangle, specifically: ", function() {
+  it("the area", function() {
+    var rectangle = {width: 6, height: 7};
+    var rectangleArea = new rectangleBits;
```

Comment:
be careful with your naming choices here. a new instance of `rectangleBits` doesn't actually represent `rectangleArea`. this could be very confusing for readers